radio/src/pulses/pxx_arm.cpp

```diff
@@ -269,6 +269,10 @@ inline void putPcmTail(uint8_t port)
 }
 #endif
 
+
+#define MAX_IERROR (0x3000)
+#define abs(x) x>0?x:-x
```

Comment:
why an `abs` macro?
side note: without any parenthesis this is *really* dangerous